tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -50,7 +50,7 @@ void DuckDBPyConnection::Initialize(py::handle &m) {
 	         "Fetch a chunk of the result as an Arrow Table following execute()", py::arg("vectors_per_chunk") = 1,
 	         py::arg("return_table") = false)
 	    .def("fetch_record_batch", &DuckDBPyConnection::FetchRecordBatchReader,
-	         "Fetch an Arrow RecordBatchReader following execute()")
+	         "Fetch an Arrow RecordBatchReader following execute()", py::arg("vectors_per_chunk") = 1)
```

Comment:
`vectors_per_chunk` seems like an unclear parameter for a user, and is also not guaranteed to contain a specific number of tuples (e.g. a vector might contain less than 1024 tuples in some cases). Perhaps we could replace this with `batch_size`?

Reply:
Perhaps an easy compromise is to change this to `approx_batch_size` and then just divide that by the vector size, so `approx_batch_size` of 1024 becomes `vectors_per_chunk=1` internally, etc.

Reply:
I'm fine with that.